VirtualList/tests/VirtualGridList-specs.js

```diff
@@ -0,0 +1,349 @@
+			/>
+		);
+
+		const expected = 24; // (4 * (720 / 240)) + 4 * 3
+		const actual = screen.getByRole('list').children.length;
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should re-render (clientHeight / itemHeight + overhang) items after changing \'clientSize\'', () => {
+		const {rerender} = render(
+			<VirtualGridList
+				clientSize={clientSize}
+				dataSize={dataSize}
+				itemRenderer={renderItem}
+				itemSize={itemSize}
+			/>
+		);
+
+		const newClientSize = {clientWidth: 1280, clientHeight: 360};
+
+		rerender(
+			<VirtualGridList
+				clientSize={newClientSize}
+				dataSize={dataSize}
+				itemRenderer={renderItem}
+				itemSize={itemSize}
+			/>
+		);
+
+		const expected = 20; // (4 * (360 / 240)) + 4 * 3
+		const actual = screen.getByRole('list').children.length;
+
+		expect(actual).toBe(expected);
+	});
+
+	test('should render only one scrollbar', () => {
+		render(
+			<VirtualGridList
+				clientSize={clientSize}
+				dataSize={dataSize}
+				direction="horizontal"
+				itemRenderer={renderItem}
+				itemSize={itemSize}
+			/>
+		);
+
+		const expected = 2; // One for the list and another for the horizontal scrollbar
+		const actual = screen.getByRole('list').parentElement.parentElement.children.length;
+
+		expect(actual).toBe(expected);
+	});
+
+	describe('Scrollbar visibility', () => {
+		test(
+			'should render both horizontal and vertical scrollbars when \'horizontalScrollbar\' and \'verticalScrollbar\' are "visible"',
+			() => {
+				render(
+					<VirtualGridList
+						clientSize={clientSize}
+						dataSize={dataSize}
+						direction="horizontal"
+						itemRenderer={renderItem}
+						itemSize={itemSize}
+						horizontalScrollbar="visible"
+						verticalScrollbar="visible"
+					/>
+				);
+
+				const VirtualGridListRoot =  screen.getByRole('list').parentElement.parentElement;
+				const verticalScrollbar = VirtualGridListRoot.children.item(1);
+				const horizontalScrollbar = VirtualGridListRoot.children.item(2);
+
+				expect(verticalScrollbar).toHaveClass("verticalScrollbar");
+				expect(horizontalScrollbar).toHaveClass("horizontalScrollbar");
+			}
+		);
+
+		test(
+			'should render only vertical scrollbar when \'verticalScrollbar\' is "visible" and \'horizontalScrollbar\' is "hidden"',
+			() => {
+				render(
+					<VirtualGridList
+						clientSize={clientSize}
+						dataSize={dataSize}
+						itemRenderer={renderItem}
+						itemSize={itemSize}
+					/>
+				);
+
+				const VirtualGridListRoot =  screen.getByRole('list').parentElement.parentElement;
+				const verticalScrollbar = VirtualGridListRoot.children.item(1);
+				const horizontalScrollbar = VirtualGridListRoot.children.item(2);
+
+				expect(verticalScrollbar).toBeInTheDocument();
+				expect(verticalScrollbar).toHaveClass("verticalScrollbar");
+				expect(horizontalScrollbar).toBeNull();
+			}
+		);
+
+		test(
+			'should not render any scrollbar when when \'horizontalScrollbar\' and \'verticalScrollbar\' are "hidden"',
```

Comment:
I found several below. Please fix.
```suggestion
			'should not render any scrollbar when \'horizontalScrollbar\' and \'verticalScrollbar\' are "hidden"',
```

Reply:
I fixed it.